### DISCUSSION TITLE
Cannot verify my school status with chsi information in China

### DISCUSSION THREAD

**Discussion by gtn1024** [community/community#53319] [GitHub Education]
I am a college student in China. My school does not provide an edu-mail account.

I could only use [chsi](https://www.chsi.com.cn/) information to verify my status, but it does not work.

Any idea to help me? I'd like to use GitHub Copilot to help me program, which is an awesome feature for me.

![image](https://user-images.githubusercontent.com/69113276/233416642-24698c38-162f-430c-ad0a-fc1b281a5afe.png)

**Comment by Spico197** [comment]
Maybe you could try to take a picture of your student ID card

**Comment by samus-aran** [comment]
Hi @gtn1024 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Reply by gtn1024** [reply]
@samus-aran However, I tried everything I had uploading to GitHub, but it does not work. I also tried to translate my chis report, which is the official website to verify ones student status, to English, fail as well. :(

<img width="912" alt="image" src="https://user-images.githubusercontent.com/69113276/233907276-d9b42508-a361-40e5-bbf4-7fca77aaa887.png">

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Comment by gtn1024** [comment]
I tried just now. And successfully verified...

I just uploaded my chis picture on it without translating to English. 😆

**Comment by fangjingmu** [comment]
点了continue没反应怎么办呀 用的流量 wifi都不好用 没用代理
![image](https://github.com/community/community/assets/150803692/fffba2b3-2ac8-40b8-9769-31ee4ca65e60)

**Reply by pixia1234** [reply]
没办法的 github烂完了
